ext/LinearSolveForwardDiff.jl

```diff
@@ -0,0 +1,88 @@
+module LinearSolveForwardDiff
+
+using LinearSolve
+using InteractiveUtils
+isdefined(Base, :get_extension) ? 
+    (import ForwardDiff; using ForwardDiff: Dual) : 
+    (import ..ForwardDiff; using ..ForwardDiff: Dual)
```

Comment:
Only 1.9+ is supported now

Reply:
I am not sure I understand. What do you mean?

Reply:
basically you dont need to do this anymore, just the first import line works